pkg/libovsdb/client.go

```diff
@@ -299,8 +299,8 @@ func (ovs OvsdbClient) MonitorCancel(jsonContext interface{}) error {
 	if err != nil {
 		return err
 	}
-	if reply.Error != "" {
-		return fmt.Errorf("Error while executing transaction: %s", reply.Error)
+	if *reply.Error != "" {
```

Comment:
It's a part of the libovsdb, but we don't have to use OperationResult here, just return an error

Reply:
will leave as is